docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -143,10 +143,12 @@ The following provides a good starting point for creating ``etc/jvm.config``:
     -Djdk.nio.maxCachedBufferSize=2000000
     -XX:+UnlockDiagnosticVMOptions
     -XX:+UseAESCTRIntrinsics
+    -XX:-G1UsePreventiveGC
```

Comment:
Can you please put it next to other GC-related properties, eg after `-XX:+ExplicitGCInvokesConcurrent`?

Also, please add a line of comment `# Disable preventive GC, see https://bugs.openjdk.org/browse/JDK-8293861 for details`

Also, please update jvm.config in RPM and product tests.

Reply:
> Can you please put it next to other GC-related properties, eg after -XX:+ExplicitGCInvokesConcurrent?

Because `-XX:-G1UsePreventiveGC` is a diagnostic option, it must be called after `-XX:+UnlockDiagnosticVMOptions`. 

So how about placing it in between `-XX:MaxRAMPercentage=80` and `-XX:G1HeapRegionSize=32M` as shown below?
```
    -server
    -Xmx16G
    -XX:InitialRAMPercentage=80
    -XX:MaxRAMPercentage=80
    -XX:+UnlockDiagnosticVMOptions
    -XX:+UseAESCTRIntrinsics
    -XX:-G1UsePreventiveGC
    -XX:G1HeapRegionSize=32M
    -XX:+ExplicitGCInvokesConcurrent
    -XX:+ExitOnOutOfMemoryError
    -XX:+HeapDumpOnOutOfMemoryError
    -XX:-OmitStackTraceInFastThrow
    -XX:ReservedCodeCacheSize=512M
    -XX:PerMethodRecompilationCutoff=10000
    -XX:PerBytecodeRecompilationCutoff=10000
    -Djdk.attach.allowAttachSelf=true
    -Djdk.nio.maxCachedBufferSize=2000000
```

Reply:
> Also, please update jvm.config in RPM and product tests.

When you mean by this sentence, do you mean all the files listed below? First line is from docker, second one is rpm, and last three is from testing/trino-product-tests-launcher.

![image](https://user-images.githubusercontent.com/25147023/207480402-ad09e658-912f-4012-936f-66504c822643.png)